apps/condo/domains/acquiring/schema/Payment.js

```diff
@@ -112,6 +109,17 @@ const Payment = new GQLListSchema('Payment', {
         delete: false,
         auth: true,
     },
+    hooks: {
+        validateInput: ({ resolvedData, context, addValidationError }) => {
+            if (!hasDvAndSenderFields(resolvedData, context, addValidationError)) return
+            const { dv } = resolvedData
+            if (dv === 1) {
+                // NOTE: version 1 specific translations. Don't optimize this logic
```

Comment:
please use explicit returns

Reply:
Linter says it's redundant and we decided not to increase warning amount :)